css/properties/content-visibility.json

```
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "content-visibility": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/content-visibility",
+          "support": {
+            "chrome": {
+              "version_added": "85"
+            },
+            "chrome_android": {
+              "version_added": "85"
+            },
+            "edge": {
+              "version_added": "85"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "71"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
```

suggestion: the Chrome status entry says WebView shipped this in 85 too

```suggestion
              "version_added": "85"
```